### DISCUSSION TITLE
Does this compatible with Pulseeffects or only Pipewire-based sound server

### DISCUSSION THREAD

**Discussion by NP-chaonay** [wwmm/easyeffects#1649] [Q&A] [answered]
Tested on Ubuntu 22.04 default sound server (PulseAudio), no device found.

**Comment by wwmm** [comment, marked as answer]
EasyEffects is only for PipeWire and PulseEffects is only for Pulseaudio. The only one being developed at this moment is EasyEffects. PulseEffects gets just fixes for critical bugs.

**Reply by NP-chaonay** [reply]
so this also indicate that PipeWire maybe really replacement for PulseAudio and PA maybe beomcing more much legacy thing (Wayland vs X11, Wayland is good replacement, however X11 has its own advantage nowaday). Thanks I hope I using PipeWire on Ubuntu 22.10 as PulseEffects have many bug on RT crashing when open dialouge and including bug that sound dont work in custom daemon.pa and require trigger output latency/buffer settings changing.
